server/views/partials/analytics.njk

```diff
@@ -8,12 +8,11 @@ h.end=i=function(){s.className=s.className.replace(RegExp(' ?'+y),'')};
 </script>
 <script>
 {# Google Analytics #}
-(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
-ga('create', 'UA-55614-6', 'auto');
+window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
```

Comment:
Could we leave a comment for our future selves and others?

Reply:
Added, it was actually from the official docs:
https://developers.google.com/analytics/devguides/collection/analyticsjs/#alternative_async_tracking_snippet